package.json

```diff
@@ -1,37 +1,107 @@
 {
   "private": true,
-  "devDependencies": {
-    "@babel/core": "^7.6.4",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/register": "^7.6.2",
-    "dotenv": "^8.1.0",
-    "eslint-plugin-compat": "^3.3.0",
-    "eslint-plugin-jasmine": "^2.10.1",
-    "globby": "^6.1.0",
-    "lerna": "^2.11.0"
+  "name": "@okta/okta-auth-js",
+  "description": "The Okta Auth SDK",
+  "version": "4.0.0",
+  "homepage": "https://github.com/okta/okta-auth-js",
+  "license": "Apache-2.0",
+  "main": "lib/server/serverIndex.js",
+  "browser": "bundles/okta-auth-js.umd.js",
+  "react-native": "lib/server/serverIndex.js",
```

Comment:
should we generate a specific bundle for react-native, use the same code base, but generate different bundles.

Reply:
I think we can use the same code we are using for Node/server. It just transpiled to regular ES3 commonJS format

Reply:
Right, it would work, my idea is since most react-native devs use esm. It would be easier for them to take advantage of toolings target to esm.

Reply:
At this point I am not publishing the original ESM source. Only a UMD webpack bundle and the babelified src. We could copy the original source as well and expose that as an entry point. Not sure exactly the best standard to follow for this. We expose several entries in our angular module: https://github.com/okta/okta-oidc-js/blob/master/packages/okta-angular/package.json